pwndbg/gdblib/events.py

```diff
@@ -136,7 +136,7 @@ def __exit__(self, *a, **kw):
 objfile_cache = dict()
 
 
-def connect(func, event_handler, name=""):
+def connect(func, event_handler, name="", receive_args=False):
```

Comment:
FWIW some GDB Python events have arguments: https://sourceware.org/gdb/onlinedocs/gdb/Events-In-Python.html#Events-In-Python